lib/src/day_view/day_view.dart

```diff
@@ -443,10 +449,6 @@ class DayViewState<T extends Object?> extends State<DayView<T>> {
 
   /// Updates data related to size of this view.
   void _updateViewDimensions() {
-    _width = widget.width ?? MediaQuery.of(context).size.width;
-
-    _timeLineWidth = widget.timeLineWidth ?? _width * 0.13;
-
```

Comment:
@ZacharyBohn Is there any reason to remove this? If we assign a width to the day view then it will not work.

Reply:
I hadn't considered when they would pass in width.  That was my bad.  It should now take that into account.